lib/internal/Magento/Framework/CompiledInterception/Test/Unit/CompiledInterceptor/_out_interceptors/ComplexItem.txt

```diff
@@ -0,0 +1,193 @@
+namespace Magento\Framework\CompiledInterception\Test\Unit\Custom\Module\Model\ComplexItem;
+
+use Magento\Framework\Config\ScopeInterface;
+use Magento\Framework\ObjectManagerInterface;
+
+/**
+ * Interceptor class for @see \Magento\Framework\CompiledInterception\Test\Unit\Custom\Module\Model\ComplexItem
+ */
+class Interceptor extends \Magento\Framework\CompiledInterception\Test\Unit\Custom\Module\Model\ComplexItem
```

Comment:
Should this apply to generated code? It is the easiest and cleanest way to avoid name collisions (Plugins can have last bit of FQCN same as intercepted class for example). Also this works this way currently and I assumed this is reasonable.